Symfony/CS/Tests/Fixer/Contrib/NoUselessReturnFixerTest.php

```diff
@@ -34,6 +34,44 @@ public function provideFixCases()
         return array(
             array(
                 '<?php
+                    function bar($baz)
+                    {
+                        if ($baz)
+                            return $this->baz();
+                        else
```

Comment:
that doesn't stop this PR being merged though - that's just something to consider for later